pandas/tests/io/test_html.py

```diff
@@ -1242,6 +1242,50 @@ def test_preserve_empty_rows(self, flavor_read_html):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_preserve_rows_with_spaces(self, flavor_read_html):
+        result = flavor_read_html(
+            StringIO(
+                """
+            <table>
+                <tr>
+                    <th>A</th>
+                    <th>B</th>
+                </tr>
+                <tr>
+                    <td>a</td>
+                    <td>b</td>
+                </tr>
+                <tr>
+                    <td>  </td>
+                    <td> </td>
+                </tr>
+            </table>
+        """
+            ),
+            skip_blank_lines=False,
+        )[0]
+        expected = DataFrame(data=[["a", "b"], ["  ", " "]], columns=["A", "B"])
+
+        tm.assert_frame_equal(result, expected)
+
+    def test_preserve_table_with_just_spaces(self, flavor_read_html):
+        result = flavor_read_html(
+            StringIO(
+                """
+            <table>
+                <tr>
+                    <td> </td>
+                </tr>
+            </table>
+        """
+            ),
+            skip_blank_lines=False,
```

Comment:
Is this necessary? If it's left as the default then what will the result be?

Reply:
I see, so removing the new argument to parse_html, I can instead pass the skip_blank_lines to be False by default to _parse function to always get space only lines as a row instead of avoiding them